.cirrus.yml

```diff
@@ -62,7 +62,7 @@ env:
 
 
 # Default timeout for each task
-timeout_in: 60m
+timeout_in: 120m
```

Comment:
Why do we bump the default? Shouldn't we only bump for e2e and unit tests?
How much slower is the new version? It worries me when the tasks now can take almost 120 minutes.

Reply:
It's not carved into stone. I am currently trying to figure out if bumping the timeout helps at all.